src/Campaigns/ValueObjects/CampaignStatus.php

```diff
@@ -24,6 +26,7 @@
  */
 class CampaignStatus extends Enum
 {
+    const ARCHIVE  = 'archive';
```

Comment:
I would prefer to rename it `archived` to keep the same name displayed on the screen. 

Also, the CSS class is already using this naming here:  https://github.com/impress-org/givewp/blob/epic/campaigns/src/Views/Components/ListTable/InterweaveSSR/styles.scss#L41

Another thing to notice is that currently this status is not being displayed correctly in the campaigns list table, it's being displayed as "Draft" and with the wrong style because of this difference between the `enum` and the `scss` file.

After renaming it, it's also necessary to refactor the `src/Campaigns/ListTable/Columns/StatusColumn.php` file and add something like it there:

```
case 'archived':
    $statusLabel = __('Archived', 'give');
    break;
```

**Campaign 6881 on the list table page:**
![image](https://github.com/user-attachments/assets/143a90ca-b85f-4c39-abf3-7a162a85a7a9)

**Campaign 6881 on the details page:**
![image](https://github.com/user-attachments/assets/bdca437f-8e00-4415-8ff8-2b0282d559e0)

Reply:
The problem with this approach is that it is not translatable. Anyway, I refactored the campaign status to `archived`, but I added support for translations as well https://github.com/impress-org/givewp/pull/7565/commits/1d32e125ad84371016ced11454a8b623534c3251